frontend/packages/core/src/Feedback/alert.tsx

```diff
@@ -4,65 +4,86 @@ import MuiErrorIcon from "@mui/icons-material/Error";
 import MuiInfoIcon from "@mui/icons-material/Info";
 import MuiWarningIcon from "@mui/icons-material/Warning";
 import type { AlertProps as MuiAlertProps } from "@mui/lab";
-import { Alert as MuiAlert, AlertTitle as MuiAlertTitle, Grid } from "@mui/material";
+import { Alert as MuiAlert, AlertTitle as MuiAlertTitle, Grid, useTheme } from "@mui/material";
 
 import styled from "../styled";
 
-const backgroundColors = {
-  error: "linear-gradient(to right, #DB3615 8px, #FDE9E7 0%)",
-  info: "linear-gradient(to right, #3548D4 8px, #EBEDFB 0%)",
-  success: "linear-gradient(to right, #1E942E 8px, #E6F7EB 0%)",
-  warning: "linear-gradient(to right, #FFCC80 8px, #FFFDE6 0%)",
-};
-
 const StyledAlert = styled(MuiAlert)<{ severity: MuiAlertProps["severity"] }>(
-  {
-    borderRadius: "8px",
-    padding: "16px",
-    paddingLeft: "24px",
-    paddingBottom: "20px",
-    color: "rgba(13, 16, 48, 0.6)",
-    fontSize: "14px",
-    overflow: "auto",
-    ".MuiAlert-icon": {
-      marginRight: "16px",
-      padding: "0",
-    },
-    ".MuiAlert-message": {
-      maxWidth: "calc(100% - 40px)",
-      width: "100%",
-      padding: "0",
-      ".MuiAlertTitle-root": {
-        marginBottom: "0",
-        color: "#0D1030",
+  () => {
+    const theme = useTheme();
```

Comment:
update pushed

Reply:
That works, may need to update the theme implementation for emotion's theme prop to use mui's, otherwise some instances seem to lack types and others throw a type error

Reply:
☝️ updated in base branch